sklearn/grid_search.py

```diff
@@ -652,6 +652,9 @@ class GridSearchCV(BaseSearchCV):
     n_jobs : int, default=1
         Number of jobs to run in parallel.
 
+        .. versionchanged:: 0.17
```

Comment:
U mean I should add to this to all n_jobs wherever applicable: Only version added is enough ?
```python
.. versionadded:: 0.17    
```